packages/theme-base/src/tokens/components/select.js

```diff
@@ -0,0 +1,28 @@
+module.exports = {
+  width: { value: '100%' },
+  transitionProperty: { value: 'background-color' },
+  transitionDuration: { value: '{time.transition.medium.value}' },
+  //variation
+  flushed: {
+    borderTop: { value: 'none' },
+    borderLeft: { value: 'none' },
+    borderRight: { value: 'none' },
+    borderRadius: { value: '0' },
+    _focus: {
+      boxShadow: { value: '0 4px 2px -2px hsl(220, 50%, 50%)' },
```

Comment:
Please see my branch for focus border color and box shadow variables

Reply:
The `boxShadow` for quiet may be a slightly different but  I will refer the color there